staging/publishing/rules.yaml

```diff
@@ -770,6 +770,10 @@ rules:
       branch: master
     - repository: api
       branch: master
+    - repository: client-go
```

Comment:
This seems like a change that should be made in an independent PR?

Reply:
Without it the test fails, so i added here https://prow.k8s.io/view/gcs/kubernetes-jenkins/pr-logs/pull/86034/pull-kubernetes-verify/1223171713915686913

Reply:
It's possible this came from adding component-base as a dependency of the Kubelet apis subdir. @liggitt @sttts WDYT? Is this change ok?

Reply:
it is because this adds k8s.io/kubelet -> k8s.io/component-base -> k8s.io/client-go

I'll defer to @sttts on whether that layering makes sense, but I think it probably does

Reply:
I think the layering makes sense, just wanted to make sure this wasn't an artifact of something else. Thanks!

Reply:
+1